src/grpc/quick/qquickgrpcsubscription.cpp

```diff
@@ -144,6 +145,8 @@ bool QQuickGrpcSubscription::subscribe()
         return false;
     }
 
+    returnValueChanged();
```

Comment:
It's better to make this before check. Pointer is rewritten by

m_returnValue = reinterpret_cast<QObject*>(returnMetaType.create());

So it's better to call change signal immediately after this line.

Reply:
Done. The only thing I did was placing it after

```
    qmlEngine(this)->setObjectOwnership(m_returnValue, QQmlEngine::CppOwnership);
```

Because i thought it might be necessary to set the ownership before signaling the change.

Reply:
For properties it's not critical. QmlEngine never gives Qml ownership for property pointer. But you approach is correct.
Thank you!